frontend/src/scenes/session-recordings/player/rrweb/canvas/canvas-plugin.ts

```diff
@@ -214,10 +206,52 @@ export const CanvasReplayerPlugin = (events: eventWithTime[]): ReplayPlugin => {
             }
 
             if (node.nodeName === 'CANVAS' && node.nodeType === 1) {
-                const el = containers.get(id) || document.createElement('img')
+                const existingContainer = containers.get(id)
+                if (existingContainer) {
+                    return // Already processed
+                }
+
+                // Create wrapper div that gets all the canvas attributes
+                const wrapper = document.createElement('div')
+                const img = document.createElement('img')
+
+                const canvasElement = node as HTMLCanvasElement
+
+                // Copy canvas attributes to wrapper div
+                if (canvasElement.id) {
+                    wrapper.id = canvasElement.id
+                }
+                if (canvasElement.className) {
+                    wrapper.className = canvasElement.className
+                }
+
+                // Copy other potentially relevant attributes to wrapper
+                const attributesToCopy = ['data-testid', 'aria-label', 'title']
+                attributesToCopy.forEach((attr) => {
+                    const value = canvasElement.getAttribute(attr)
+                    if (value) {
+                        wrapper.setAttribute(attr, value)
+                    }
+                })
```

Comment:
why allowlist only some attributes? we don't know which will be important to customers or to display

id and class are iirc just attributes, so we can "just" copy all attributes and we'd get id and class too